### PR TITLE
Get rid of redundant arity field from op defs.

### PR DESCRIPTION
**Pull request by happi**
Also
 Reorder and renumber ops.
 Fix bb_end for abort and exit.
 Replace atomic with [] for consistency

### CODE REVIEW

File: src/aeb_fate_generate_ops.erl [hunk 1]
```diff
@@ -349,21 +347,22 @@ gen_defines(#{opname := Name, opcode := OpCode}) ->
...
+               [] -> io_lib:format("~w", [OpName]);
                ArgTypes  ->
                     io_lib:format("{~w, ~s}", [OpName, expand_types(ArgTypes)])
            end,
     ConstructorType = atom_to_list(Constructor) ++ "/" ++ io_lib:format("~w", [Args]),
 
     [#{ opname            => OpName
       , opcode            => OpCode
-      , args              => Args
+      , arity             => Args
```
Comment (gorillainduction) [inline src/aeb_fate_generate_ops.erl]: ...and here

File: src/aeb_fate_generate_ops.erl [hunk 2]
```diff
@@ -669,7 +668,7 @@ generate_documentation(Filename) ->
 
 gen_doc(#{ opname            := Name
          , opcode            := OpCode
-         , args              := _Args
+         , arity             := _Args
```
Comment (gorillainduction) [inline src/aeb_fate_generate_ops.erl]: ...and here.

File: src/aeb_fate_generate_ops.erl [hunk 3]
```diff
@@ -349,21 +347,22 @@ gen_defines(#{opname := Name, opcode := OpCode}) ->
 
 gen([]) ->
     [];
-gen([{OpName, OpCode, Args, EndBB, Gas, FateFormat, Constructor, Doc} | Rest]) ->
+gen([{OpName, OpCode, EndBB, Gas, FateFormat, Constructor, Doc} | Rest]) ->
+    Args = length(FateFormat),
```
Comment (gorillainduction) [inline src/aeb_fate_generate_ops.erl]: Why not use `Arity` instead of `Args` here?

Comment (happi) [reply src/aeb_fate_generate_ops.erl]: I'll fix when I do some other stuff... soon.

**Pull request merged**